tools/oozie/oozie_simple_test/workflow.xml

```diff
@@ -0,0 +1,51 @@
+<workflow-app xmlns='uri:oozie:workflow:0.3' name="test">
+   <start to="run_test"/>
+   <action name="run_test">
+      <shell xmlns="uri:oozie:shell-action:0.1">
+         <job-tracker>${jobTracker}</job-tracker>
+         <name-node>${nameNode}</name-node>
+         <configuration>
+            <property>
+               <name>mapred.job.queue.name</name>
+               <value>${queueName}</value>
+            </property>
+            <property>
+               <name>oozie.launcher.mapreduce.map.memory.mb</name>
```

Comment:
There is already a default in oozie-site.xml it mirros same settings as mapred-site.xml, but prepends oozie.launcher
https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/templates/default/ooz_oozie-site.xml.erb#L357

https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/templates/default/ooz_oozie-site.xml.erb#L367